contracts/modules/assessment/Assessment.sol

```diff
@@ -2,515 +2,523 @@
 
 pragma solidity ^0.8.28;
 
-import "@openzeppelin/contracts-v4/utils/cryptography/MerkleProof.sol";
+import {EnumerableSet} from "@openzeppelin/contracts-v4/utils/structs/EnumerableSet.sol";
 
-import "../../abstract/MasterAwareV2.sol";
-import "../../interfaces/IAssessment.sol";
-import "../../interfaces/IMemberRoles.sol";
-import "../../interfaces/INXMToken.sol";
-import "../../interfaces/ITokenController.sol";
-import "../../interfaces/IRamm.sol";
-import "../../libraries/Math.sol";
-import "../../libraries/SafeUintCast.sol";
+import {Multicall} from "../../abstract/Multicall.sol";
+import {IAssessment} from "../../interfaces/IAssessment.sol";
+import {RegistryAware, C_GOVERNOR, C_COVER_PRODUCTS, C_CLAIMS, C_ASSESSMENT} from "../../abstract/RegistryAware.sol";
+import {SafeUintCast} from "../../libraries/SafeUintCast.sol";
```

Comment:
Please import the files directly as we do through the repo in other contracts. If there are name conflicts try to rename or move the conflicting items inside the interface (as opposed to keeping them at top level).